test_scripts/API/VehicleData/GpsShiftSupport/commonGpsShift.lua

```diff
@@ -0,0 +1,124 @@
+    actual = false,
+    satellites = 8,
+    dimension = "2D",
+    altitude = 7.7,
+    heading = 173.99,
+    speed =  2.78
+}
+
+--[[ Functions ]]
+function m.pTUpdateFunc(tbl)
+    tbl.policy_table.app_policies[config.application1.registerAppInterfaceParams.fullAppID].groups = {"Base-4", "Location-1"}
+end
+
+function m.checkShifted(data, pShiftValue)
+    if pShiftValue == nil and data ~= nil then
+        return false, "Vehicle data contains unexpected shifted parameter"
+    end
+    return true
+end
+
+function m.getVehicleData(pShiftValue)
+    m.gpsParams.shifted = pShiftValue
+    local cid = m.getMobileSession():SendRPC("GetVehicleData", { gps = true })
+    EXPECT_HMICALL("VehicleInfo.GetVehicleData", { gps = true })
+    :Do(function(_, data)
+        m.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", { gps = m.gpsParams })
+    end)
+    m.getMobileSession():ExpectResponse(cid, { success = true, resultCode = "SUCCESS", gps = m.gpsParams })
+    :ValidIf(function(_, data)
+        return m.checkShifted(data.payload.gps.shifted, pShiftValue)
+    end)
+end
+
+function m.subscribeVehicleData()
+    local gpsResponseData = {
+        dataType = "VEHICLEDATA_GPS",
+        resultCode = "SUCCESS"
+      }
+    local cid = m.getMobileSession():SendRPC("SubscribeVehicleData", { gps = true })
+    EXPECT_HMICALL("VehicleInfo.SubscribeVehicleData", { gps = true })
+    :Do(function(_, data)
+        m.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", { gps = gpsResponseData })
+    end)
+    m.getMobileSession():ExpectResponse(cid, { success = true, resultCode = "SUCCESS", gps = gpsResponseData })
+end
+
+function m.sendOnVehicleData(pShiftValue)
+    m.gpsParams.shifted = pShiftValue
+    m.getHMIConnection():SendNotification("VehicleInfo.OnVehicleData", { gps = m.gpsParams })
+    m.getMobileSession():ExpectNotification("OnVehicleData", { gps = m.gpsParams })
+    :ValidIf(function(_, data)
+        return m.checkShifted(data.payload.gps.shifted, pShiftValue)
+    end)
+end
+
+function m.getInteriorVehicleData(pShiftValue, pIsSubscribed)
+    if not pIsSubscribed then pIsSubscribed = false end
+    m.radioData.radioControlData.sisData.stationLocation.shifted = pShiftValue
+    local cid = m.getMobileSession():SendRPC("GetInteriorVehicleData", {
+        moduleType = "RADIO",
+        subscribe = true
+    })
+    EXPECT_HMICALL("RC.GetInteriorVehicleData", {
+        moduleType = "RADIO",
+        subscribe = true
+      })
+    :Do(function(_, data)
+        m.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", {
+            moduleData = m.radioData,
+            isSubscribed = pIsSubscribed
+          })
+    end)
+    m.getMobileSession():ExpectResponse(cid, { success = true, resultCode = "SUCCESS",
+      isSubscribed = pIsSubscribed,
+      moduleData = m.radioData
+    })
+    :ValidIf(function(_, data)
+        return m.checkShifted(data.payload.moduleData.shifted, pShiftValue)
```

Comment:
@TMyza Please check if `data.payload.moduleData.shifted` is a correct path for `shifted` parameter